gordo/machine/dataset/data_provider/file_type.py

```diff
@@ -100,10 +99,6 @@ def __init__(self, time_series_columns: TimeSeriesColumns):
     def read_df(self, f: BinaryIO) -> pd.DataFrame:
         columns = self.time_series_columns.columns
         datetime_column = self.time_series_columns.datetime_column
-        pf = pa.ParquetFile(f)
-        table = pf.read(columns)
-        df = table.to_pandas()
-        df[datetime_column] = df[datetime_column].apply(
-            lambda v: pd.to_datetime(v, utc=True)
```

Comment:
As @epa095 points out, if you remove the pd.to_datetime from the read_df in the ParquetFileType, and put it into the read_df in ncs_reader in stead, you should remove it from the CsvFileType. Alternatively, just add the pd.to_datetime statement to CsvFileType again, and remove it from read_df.

Reply:
moved it to ParqutFileType.